sdk/core/azure-core/test/ut/azure_libcurl_core_main_test.cpp

```diff
@@ -52,12 +58,53 @@ namespace Azure { namespace Core { namespace Test {
       EXPECT_TRUE(session->IsEOF());
       EXPECT_TRUE(session->m_keepAlive);
       EXPECT_FALSE(session->m_connectionUpgraded);
+      t1 = high_resolution_clock::now();
+    }
+    // here the session is destroyed and the connection is moved to the pool
+    // the same destructor also makes a call to start the cleanup thread
+    // which will sleep for DefaultCleanerIntervalMilliseconds then loop through the connections
+    // in the pool and check for the ones that are expired(DefaultConnectionExpiredMilliseconds) and
+    // remove them which will be the case here if we wait long enough. the test is flaky due to the
+    // fact that tests in the CI pipeline might take longer than 90 sec to execute thus the cleanup
+    // thread strikes. to have this test be predictable we should let things run to completion and
+    // then check the pool size thus the sleep below plus another second to let the for loop in the
+    // thread do its thing.
+
+    auto t2 = high_resolution_clock::now();
+
+    /* Getting number of milliseconds as an integer. */
+    auto ms_int = duration_cast<milliseconds>(t2 - t1);
```

Comment:
This local variable seems to be unsused. Why are we getting the ms both as an int and as a double?

Reply:
removed